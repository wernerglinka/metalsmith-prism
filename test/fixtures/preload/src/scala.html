<code class="language-scala">
  val hello: String = "Hello world!"
</code>
