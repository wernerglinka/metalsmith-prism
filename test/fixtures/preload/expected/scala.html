<code class="language-scala">
  <span class="token keyword">val</span> hello<span class="token operator">:</span> <span class="token builtin">String</span> <span class="token operator">=</span> <span class="token string">"Hello world!"</span>
</code>
